pymeasure/instruments/agilent/agilent33500.py

```diff
@@ -226,16 +547,18 @@ def beep(self):
     )
 
     output = Instrument.control(
-        "OUTP?", "OUTP %d",
+        "OUTP?",
+        "OUTP %d",
         """ A boolean property that turns on (True, 'on') or off (False, 'off')
         the output of the function generator. Can be set. """,
         validator=strict_discrete_set,
         map_values=True,
-        values={True: 1, 'on': 1, 'ON': 1, False: 0, 'off': 0, 'OFF': 0},
+        values={True: 1, "on": 1, "ON": 1, False: 0, "off": 0, "OFF": 0},
```

Comment:
I see many changes involving replacement of single quote with double quote and viceversa. Are these changes really needed?

Reply:
Before committing I reformated the file with black. The replacements are not needed as in they don't break anything either way. While black is PEP 8 compliant, it is true it has its own style. Let me know what style the project prefers and I can change the device and tests to it.

Reply:
The reformatting (also splitting/merging lines, changing number formatting) makes it hard to see the actual implementation details.
In fact, we have black style configuration in this project (I personally do not use black, but it was recently added).

Here a quote from our [documentation](https://pymeasure.readthedocs.io/en/latest/dev/coding_standards.html?highlight=black#python-style-guides), how to use it (for better code review):

> "It is allowed but not required to use the [black](https://black.readthedocs.io/en/stable/) code formatter. Do only format files that you are actively working on and make a dedicated commit for the initial formatting when working on an existing file that does not adhere to the black code style to facilitate code review. If there are conflicts between black’s output and flake8 (especially related to [E203](https://www.flake8rules.com/rules/E203.html)), flake8 takes precedence. Use #noqa : E203 to disable E203 warnings for a specific line if appropriate."

Reply:
I proposed (#767) to use https://github.com/akaihola/darker instead of black to limit the fallout from the formatting -- could you maybe retry with that tool, instead, and report back your experience?

Reply:
I guess I'm not clear on what the recommended action is. Please advise

Reply:
To me, you can leave the changes done by black.